src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -183,7 +183,8 @@ def do_POST(self):
             assert_equal(event['subject'], record['s3']['object']['key'])
             assert_equal(parser.parse(event['time']), parser.parse(record['eventTime']))
         log.info('HTTP Server received event: %s', str(body))
-        self.server.append(json.loads(body))
+        if body != b'':
```

Comment:
why do we have an empty message? and how can this happen if content length is not zero (see line 169)?